btr-web/btr-main-app/pages/reviewConfirm.vue

```diff
@@ -32,4 +38,5 @@ import { storeToRefs } from 'pinia'
 
 const significantIndividuals = useSignificantIndividuals()
 const { currentSIFiling } = storeToRefs(significantIndividuals)
+const account = storeToRefs(useBcrosAccount())
```

Comment:
when using storeToRefs I'd prefer it if we dereference the variables directly like: `const { userFullName } = storeToRefs(useBcrosAccount())` to avoid confusion (the response only contains the ref/reactive properties, no actions)